docker-deploy/backend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rockylinux/rockylinux
+MAINTAINER Nguyen Van Hieu <ngvanhieu112233@gmail.com>
+
+RUN yum update -y && yum install -y \
+  glibc-langpack-en
+
+COPY jre8-latest.rpm /opt/
```

Comment:
Do not commit the big RPM file to git.
If it needs to be downloaded separately before, put those instructions to Readme with the full download link https://www.oracle.com/java/technologies/javase-jre8-downloads.html

Reply:
Thanks, I fixed it